terasoluna-tourreservation-web/pom.xml

```diff
@@ -46,13 +47,12 @@
         <!-- == Begin TourReservation == -->
         <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>terasoluna-tourreservation-env</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
> But original 5.0.x/domain project depends on env project and 5.0.x project cant work if env scope change for test in domain pom .

Please investigate the cause.
You should not make lazy corrections, please respect the thought of the blank project.